config/ci.json

```diff
@@ -31,5 +31,9 @@
   },
   "fixer": {
     "accessKey": "UNDEFINED_FIXER_ACCESS_KEY"
+  },
+  "twoFactorAuth": {
+    "secretKey": "guineapigs",
+    "cipher": "DES"
```

Comment:
It seems to me that we should just pick the strongest encryption available for us and stick with it. Especially because if you change this, you'll need to invalidate all existing tokens which requires a DB migration.

Is there any value in making the cipher customizable that I'm missing?

Reply:
Just a further layer of security I figured, if someone can't guess which one you're using.

Reply:
As of today, they can guess we'll be using `DES`.

Reply:
We aren't 😃